.github/workflows/CI.yml

```diff
@@ -49,7 +49,7 @@ jobs:
       - name: Pull and Run Docker
         run: |
           plugin=`ls build/distributions/*.zip`
-          version=1.3.0-SNAPSHOT
+          version=1.3.0
```

Comment:
Does infra team change the name convention to remove "SNAPSHOT" ? Not sure if `1.3.0` docker built from the latest code or not.

Reply:
It is. I confirmed with @peterzhuamazon that's the case as well. They are generated from the distribution builds which are continuously building and pulling the latest changes based on the component branches in the manifest files. You can see the tags here: https://hub.docker.com/r/opensearchstaging/opensearch/tags

Reply:
Thanks, should all plugins change to 1.3.0 now?

Reply:
I think it's a good strategy to pick up the latest distribution builds, yes